### ISSUE TITLE
Deprecated NPM package doesn't shows warning on install

### ISSUE THREAD

**Issue by g4s8** [artipie/npm-adapter#174]
After running deprecate on NPM package, the message and package appears in metadata file in registry, but on install `npm` doesn't shows a warning like promised in https://docs.npmjs.com/cli/v6/commands/npm-deprecate

It can be tested with these steps:
 1. Build artipie docker and run it from `artipie/examples/npm` similar to `run.sh`
 2. Go to `sample-npm-project` dir and upload it using `npm --registry=http://localhost:8080 publish`
 3. Run `npm --registry=http://localhost:8080 deprecate sample-npm-project@1.0.0`
 4. Go to `../sample-consumer/` and run `npm --registry=http://localhost:8080 install`

Expected: `npm` shows a warning about deprecated package
Actual: installed without warning

**Comment by olenagerasimova** [comment]
@g4s8 @genryxy hm, as far as I understood from npm docs massage is mandatory, so may be npm simply does not add `deprecated` tag to the metadata in this case?

**Comment by genryxy** [comment]
@olenagerasimova Do you mean to add `deprecated` tag not only to `meta.json` file but also for `package.json` files from archives?

**Comment by olenagerasimova** [comment]
@genryxy no, `package.json` should not contain `deprecate` field, only `meta.json` should. On `deprecate` `npm` sends us PUT request with `meta.json` in the body, where necessary `deprecate` fields were added (check DeprecateSliceTest as an example). So, may be, in the case with empty message in `deprecate` command, `npm` simply does nothing?

**Comment by genryxy** [comment]
@g4s8 I added IT to show that warn is shown. Probably it is necessary to publish package from `../sample-consumer/` directory before run `npm --registry=http://localhost:8080 install` because `simple-npm-project` is dependency of `sample-consumer`.

**Comment by genryxy** [comment]
Here was an attempt to add IT case for reproducing this problem. Some changes were introduced to the source code to eliminate this problem. After that added IT case worked in a proper way (e.g. shew warning message during installation). 
Although, after local check the message was not shown. Probably there was a mistake during local check or something else.
